<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="VXBLRWKS03" name="com.qa.tests.RegisterTests" tests="7" failures="1" timestamp="2023-04-19T17:34:42 IST" time="72.872" errors="0">
  <testcase name="step1withoutTC" time="10.474" classname="com.qa.tests.RegisterTests"/>
  <system-out/>
  <testcase name="step1withTC" time="5.541" classname="com.qa.tests.RegisterTests"/>
  <system-out/>
  <testcase name="step2validmail" time="6.021" classname="com.qa.tests.RegisterTests"/>
  <system-out/>
  <testcase name="step3invalidpassword" time="8.887" classname="com.qa.tests.RegisterTests"/>
  <system-out/>
  <testcase name="step3notfollowingpasswordrequirement" time="16.030" classname="com.qa.tests.RegisterTests">
    <failure type="java.lang.AssertionError" message="expected [At least 8 character in length,At least one lowercase,At least one uppercase letter] but found [null]">
      <![CDATA[java.lang.AssertionError: expected [At least 8 character in length,At least one lowercase,At least one uppercase letter] but found [null]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.qa.tests.RegisterTests.step3notfollowingpasswordrequirement(RegisterTests.java:87)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- step3notfollowingpasswordrequirement -->
  <system-out/>
  <testcase name="step3validpassword" time="25.919" classname="com.qa.tests.RegisterTests"/>
  <system-out/>
  <testcase name="step2inavalidmail" classname="com.qa.tests.RegisterTests">
    <skipped/>
  </testcase> <!-- step2inavalidmail -->
  <system-out/>
</testsuite> <!-- com.qa.tests.RegisterTests -->
